lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/scheduler/RandomTaskletScheduler.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.vortex.driver.scheduler;
+
+import org.apache.reef.vortex.driver.tasklet.Tasklet;
+import org.apache.reef.vortex.driver.tasklet.TaskletExecutor;
+
+import javax.inject.Inject;
+import java.util.Collection;
+import java.util.Random;
+
+/**
+ * Randomly select an executor.
+ */
+public class RandomTaskletScheduler implements TaskletScheduler {
+  @Inject
+  public RandomTaskletScheduler() {
+  }
+
+  @Override
+  public TaskletExecutor schedule(final Collection<TaskletExecutor> availableExecutors, final Tasklet tasklet) {
```

Comment:
Since we are still in the stage of exploring ways to optimize the data structure for scheduling, for now I will just remove the scheduler package altogether and embed the random scheduler into `RunningExecutors` class

Once we have better understanding of key metrics(e.g. resource reliability, queue size, data locality) for making scheduling decisions we can revisit this as a separate JIRA issue. For example we can maintain data structures in `RunningExecutors` in a way that enables fast scheduling using indices on those metrics.